docs/pullsubscription/README.md

```diff
@@ -40,10 +37,10 @@ Pub/Sub topic, such as
    kubectl apply --filename pullsubscription.yaml
    ```
 
-1. [Optional] If not using GKE, or want to use a Pub/Sub topic from another
-   project, uncomment and replace the
-   [`MY_PROJECT` placeholder](https://cloud.google.com/resource-manager/docs/creating-managing-projects)
-   in [`pullsubscription.yaml`](pullsubscription.yaml) and apply it.
+1. **[Optional]** If you are not using GKE, or want to use a Pub/Sub topic from another
+   project, uncomment and replace the `MY_PROJECT` placeholder in [`pullsubscription.yaml`](pullsubscription.yaml) and apply it. 
+   Note that the Service Account during the [installation](../install/README.md) step should be able to manage
+   [multiple projects](../install/managing-multiple-projects.md).
```

Comment:
It's better to also mention this in the `../install/README.md`.

Reply:
Agree! Done, thanks!